src/modules/tas_studio/editor/utils.rs

```diff
@@ -24,6 +24,31 @@ pub trait FrameBulkExt {
 
     /// Returns a mutable reference to the yawspeed stored in the framebulk, if any.
     fn yawspeed_mut(&mut self) -> Option<&mut f32>;
+
+    /// Return a reference to the starting yaw offset, target yaw offset, and acceleration stored in
+    /// the framebulk, if any.
+    fn max_accel_yaw_offset(&self) -> Option<MaxAccelOffsetValues>;
+
+    // Return a mutable reference to the starting yaw offset, target yaw offset, acceleration,
+    // and original yaw field value stored in the framebulk, if any.
+    fn max_accel_yaw_offset_mut(&mut self) -> Option<MaxAccelOffsetValuesMut>;
+}
+
+#[allow(dead_code)]
```

Comment:
```suggestion
```
I think it's used?

Reply:
`yaw` and `count` members are not used. But they are used in the mutable struct.

Reply:
Could you put this on those members?